README.md

```diff
@@ -22,7 +22,7 @@ pip install submit4dn --upgrade
 
 ### Troubleshooting
 
-This package is not supported on older Python versions and is supported and tested for versions 3.6 - 3.9.  It may work with other python versions but your mileage may vary.
+This package is not supported on older Python versions and is supported and tested for versions 7 - 3.9.  It may work with other python versions but your mileage may vary.
```

Comment:
```suggestion
This package is not supported on older Python versions and is supported and tested for versions 3.7 - 3.9.  It may work with other python versions but your mileage may vary.
```